workshops/management/commands/report_incomplete_instructors.py

```diff
@@ -1,41 +0,0 @@
-import datetime
```

Comment:
No, we had some files that were getting data from API, but they were removed.

`report_incomplete_instructors.py` was quite out of date (the logic wasn't nearly as good as our `Instructor issues` view).